Dockerfile

```diff
@@ -37,4 +40,11 @@ RUN echo -e "source /opt/ros/noetic/setup.bash\n" \
 
 # build deps
 RUN source ~/.profile && cd amrl_msgs && make -j
-RUN source ~/.profile && cd ut_automata && make -j
+RUN source ~/.profile && cd ut_automata && make 
```

Comment:
can you elaborate on what the error you got was? I'm curious why it affected ut_automata and not amrl_msgs

Reply:
I was getting linker errors on the build. I believe it was trying to link an object that wasn't finished compiling yet. It only occurs on my Macbook, but runs fine on my Linux box. It was throwing the same error when I was building a separate Dockerfile before I saw the original Dockerfile PR. May be worthwhile to take the `-j` off of `amrl_msg` as well for safety.

Reply:
alright go ahead and remove it from there as well, then we can merge it in